packages/arb-token-bridge-ui/src/components/App/AppContext.tsx

```diff
@@ -111,11 +163,28 @@ export const useAppContextActions = (dispatchOverride?: Dispatch<Action>) => {
     dispatch({ type: 'layout.set_txhistory_panel_visible', payload: false })
   }
 
+  const openPreferences = () => {
```

Comment:
We discussed in priv, but for visibility we will be removing those because it's merge issue

Reply:
Removed